### ISSUE TITLE
Audio playback encountered exception

### ISSUE THREAD

**Issue by RorickTech** [Darkside138/DiscordSoundboard#115]
I have used this soundbot without issue for a since the release of BETA 2.2.2, and recently rebuilt the windows VM it was running on. Fresh install, fresh everything. It loads fine, the web interface works, the bot enters my voice channel when called by command or web interface, BUT plays no sound. 

CMD log shows this error. (See attached image)
![error](https://user-images.githubusercontent.com/29452502/66157439-a4acde80-e5d8-11e9-9d18-7b384943e666.PNG)

Anyone else experience this? It appears I am missing some critical libraries, but I am not sure where to acquire these.

**Comment by RorickTech** [comment]
And since I am serious about getting this working again, here are some more system details that may be of use:

Windows 10 Pro (licensed) VM running on ESXI
Windows 10 build 18362
Java's installed:
32 bit JRE 8.0.22.10
32 bit JDK 8.0.22.10

I read in a previous issue that 64 bit Java installs caused issues with this bot so I removed them, but the issue remains

**Comment by RorickTech** [comment]
*fastest ticket in the west*

Solved. I was a java version issue. I had installed Java as I always have using Ninite.com (amazing tool if you don't know of it), but their java installations appear to have changed. 

Ninite installed OpenAdopt java something something... I don't know what all that is, so uninstalled all javas, and installed the latest 32x bit version direct from javas/oracles website, installed that.
(From the official website, I got just a vanilla java install with no frills/developer kits/or anything else.)

BOOM! working like a champ again.

Thank you Darkside138 for this awesome app!
